cmd/po-docgen/api.go

```diff
@@ -26,12 +26,20 @@ import (
 )
 
 const (
-	firstParagraph = `<br>
-<div class="alert alert-info" role="alert">
-    <i class="fa fa-exclamation-triangle"></i><b> Note:</b> Starting with v0.39.0, Prometheus Operator requires use of Kubernetes v1.16.x and up.
-</div>
-
-# API Docs
+	firstParagraph = `---
+title: "API"
+description: "Generated API docs for the Prometheus Operator"
+lead: ""
+date: 2021-03-08T08:49:31+00:00
+lastmod: 2021-03-08T08:49:31+00:00
```

Comment:
Hmm, this would need to be either adjusted to the current date or removed. Embedding last mod as a static date doesn't make much sense to me.

Reply:
Sure, we can do that. I don't think it's used anyway.